src/modules/series-stats/server/procedures/results-series/index.ts

```
@@ -0,0 +1,169 @@
+      throw new TRPCError({
+        code: "BAD_GATEWAY",
+        message: payload.error.message,
+      });
+    }
+
+    console.log(payload.data);
+    // TODO transform data before return
+    return {
+      payload: payload.data,
+      total: payload.data.length,
+    };
```

Biome (2.3.13)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)